ironfish/src/wallet/account/account.ts

```diff
@@ -122,22 +122,30 @@ export class Account {
     assetId: Buffer,
     options?: {
       confirmations?: number
+      reverse?: boolean
     },
   ): AsyncGenerator<DecryptedNoteValue> {
     const head = await this.getHead()
     if (!head) {
       return
     }
-
     const confirmations = options?.confirmations ?? 0
-
     const maxConfirmedSequence = Math.max(head.sequence - confirmations, GENESIS_BLOCK_SEQUENCE)
 
-    for await (const decryptedNote of this.walletDb.loadUnspentNotes(
+    for await (const unspentNoteHash of this.walletDb.loadValueToUnspentNoteHashes(
       this,
       assetId,
-      maxConfirmedSequence,
+      options?.reverse ?? false,
```

Comment:
why are you using `?? false`? It looks like `loadValueToUnspentNoteHashes` will default to false if it's undefined.

Reply:
Made this change